### PR TITLE
find seqs from EndBlocks

### PR DESCRIPTION
**Pull request by songwongtp**
find seqs in endblocks when seqs cannot be found in any txs (for bandchain endblock event)

### CODE REVIEW

File: relayer/naive-strategy.go [hunk 1]
```diff
@@ -567,6 +585,59 @@ func (nrs *NaiveStrategy) RelayPackets(src, dst *Chain, sp *RelaySequences) erro
...
 
+// relayEndBlockPacketFromSequences relays packets in endblock with
+// given seqs on src, and returns recvPacketmsgs, timeoutPacketmsgs and error
+func relayEndBlockPacketFromSequence(src, dst *Chain, seq uint64) ([]sdk.Msg, []sdk.Msg, error) {
+	blockResults, err := src.QueryBlockResults(src.MustGetLatestLightHeight(), 1, 1000, rcvPacketQuery(src.PathEnd.ChannelID, int(seq)))
+	switch {
+	case err != nil:
+		return nil, nil, err
+	case len(blockResults) == 0:
+		// NOTE: search the given seq in endblock instead
```
Comment (taobun) [inline relayer/naive-strategy.go]: ?

**Review by taobun — CHANGES_REQUESTED** [review]

File: relayer/naive-strategy.go (hunk 1 above)
Comment (songwongtp) [reply relayer/naive-strategy.go]: whoops. need to remove the comment

File: relayer/naive-strategy.go [hunk 2]
```diff
@@ -567,6 +585,59 @@ func (nrs *NaiveStrategy) RelayPackets(src, dst *Chain, sp *RelaySequences) erro
...
+	}
+
+	rcvPackets, timeoutPackets, err := relayPacketsFromResultBlockResults(src, dst, seq, blockResults[0])
+	switch {
+	case err != nil:
+		return nil, nil, err
+	case len(rcvPackets) == 0 && len(timeoutPackets) == 0:
+		return nil, nil, fmt.Errorf("no relay msgs created from query response")
+	case len(rcvPackets)+len(timeoutPackets) > 1:
+		return nil, nil, fmt.Errorf("more than one relay msg found in tx query")
```
Comment (taobun) [inline relayer/naive-strategy.go]: tx?

**Review by taobun — CHANGES_REQUESTED** [review]

File: relayer/naive-strategy.go (hunk 1 above)
Comment (songwongtp) [reply relayer/naive-strategy.go]: see a11cd6f

File: relayer/naive-strategy.go (hunk 2 above)
Comment (songwongtp) [reply relayer/naive-strategy.go]: a11cd6f

**Review by taobun — APPROVED** [review]

**Pull request merged**